images/manageiq-base/Dockerfile

```diff
@@ -35,9 +35,10 @@ COPY container-assets/create_local_yum_repo.sh /
 
 RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
 
-RUN dnf -y --disableplugin=subscription-manager install \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \
+RUN if [ ${ARCH} != "s390x" ] ; then dnf -y --disableplugin=subscription-manager install \
+      http://mirror.centos.org/centos/8.3.2011/BaseOS/${ARCH}/os/Packages/centos-linux-repos-8-2.el8.noarch.rpm \
```

Comment:
These shouldn't be reverted

Reply:
These repos does not work for s390x so added a check for non-s390x arch

Reply:
The conditional is fine, but you reverted it from centos stream to centos 8.3